### ISSUE TITLE
Remove redundant aria-hidden from image when no alt text is set

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-languagePicker#126]
### Subject of the enhancement

To align with core change of using `alt` instead of `aria-label` - https://github.com/adaptlearning/adapt-contrib-core/issues/822

Testing confirmed an empty alt tag (`alt=""`) is widely supported by screen readers / browsers to ignore an image without the need for `aria-hidden`.

[Applies to image, line 10](https://github.com/adaptlearning/adapt-contrib-languagePicker/blob/a9406d951510ea151204843561096ad2e3028ab4/templates/languagePickerView.hbs#L10).
